actions/class.RestQtiTests.php

```diff
@@ -119,6 +124,36 @@ public function import()
         }
     }
 
+    protected function getItemClassUri(): ?string
+    {
+        $itemClassUri = $this->getRequestParameter(self::ITEM_CLASS_URI);
+        if(!is_string($itemClassUri)) {
+            return null;
+        }
+
+        return $itemClassUri;
```

Comment:
The method `getRequestParameter` is deprecated and you can simplify your call by:

```suggestion
    protected function getItemClassUri(): ?string
    {
        return $this->getPostParameter(self::ITEM_CLASS_URI, null);
```

Reply:
fixed